docs/manual/developer/03_creating_content.md

```diff
@@ -434,10 +434,15 @@ related to this control.
 * R3 can be automatically scanned by SCAP but unfortunately we don’t have any
 rules implemented yet.
 
-For each control we will add the `automated` key, which describes whether the
-control requirement can be automated by SCAP and scanning. Possible values are:
-`yes`, `no`, `partially`. The `automated` key is just for informational purposes
-and does not have any impact on the processing.
+For each control we will add the `status` key, which describes the current
+implementation status of the control. For instance, if the control requirement
+can be automated by SCAP and scanning, the status will be `automated`.
+The `status` key is just for informational purposes and does not have any
+impact on the processing.
+
+The `automated` key was avaliable, which defines whether a control can be
+checked for using SCAP atomation. However, the `status` key is preferred as
+it provides more information.
```

Comment:
I suggest to alter the text so it is more obvious that we are dealing with a deprecation message and that if one doesn't recall the `automated` key, there is nothing to see here.

```suggestion
The  `status` key deprecates the `automated` key -
`automated: yes` translates to `status: automated`, and so on.
The `status` key is preferred as it it is capable to reflect the control state more accurately.
```